### PR TITLE
Feature/authorise agent

### PR DESCRIPTION
**Pull request by kitanoyoru**

### CODE REVIEW

File: platform-dependent-components/problem-solver/py/requirements.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 py-sc-kpm==0.2.0
+py-sc-client==0.3.0
+requests==2.31.0
+wikipedia
```
Comment (MksmOrlov) [inline platform-dependent-components/problem-solver/py/requirements.txt:4]: whyyyy?

File: sc-models/user-interface/kb/core/section_subjdomain_informational_structures_not_sc_code/nl_files/explanation_for_class_of_syntactically_equivalent_information_structures.html
Comment (MksmOrlov) [inline sc-models/user-interface/kb/core/section_subjdomain_informational_structures_not_sc_code/nl_files/explanation_for_class_of_syntactically_equivalent_information_structures.html:1]: do not format other files. It is fucking impossible to review PR with 4k editions

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
Idl if we nned this agent in this representation. Need to discuss

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
Change target branch to `demo`, please

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
And fix unresolved comments please
image

File: platform-dependent-components/problem-solver/py/modules/authoriseModule/AuthoriseUserAgent.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    get_edge
+)
+from sc_kpm.utils.action_utils import (
+    create_action_answer,
+    finish_action_with_status,
+    get_action_arguments,
+    get_element_by_role_relation
+)
+from wikipedia import *
+from random import *
```
[the remark this answers is not in the document]
Comment (kitanoyoru) [reply platform-dependent-components/problem-solver/py/modules/authoriseModule/AuthoriseUserAgent.py]: @MksmOrlov removed

File: platform-dependent-components/problem-solver/py/requirements.txt (hunk 1 above)
Comment (kitanoyoru) [reply platform-dependent-components/problem-solver/py/requirements.txt]: @MksmOrlov were unused dependecies

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Comment by MksmOrlov** [comment]
Moved to the https://github.com/ostis-ai/ostis-metasystem/pull/49

**Pull request closed without merging**